### PR TITLE
Fix #305676: fix barline translations

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
* Fix mismatch in barline translations for master palette
* Fix barline translations in workspaces palettes

### CODE REVIEW

File: libmscore/sym.cpp [hunk 1]
```diff
@@ -3281,9 +3281,9 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
       "Heavy barline",
       "Heavy double barline",
       "Reverse final barline",
-      "Short barline",
+      QT_TRANSLATE_NOOP("symUserNames", "Short barline"),
```
Comment (Jojo-Schmitz) [inline libmscore/sym.cpp:3284]: not really needed, as duplicated in menus.cpp

File: libmscore/sym.cpp [hunk 2]
```diff
@@ -3281,9 +3281,9 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
...
       QT_TRANSLATE_NOOP("symUserNames", "Single barline"),
-      "Tick barline",
+      QT_TRANSLATE_NOOP("symUserNames", "Tick barline"),
```
Comment (Jojo-Schmitz) [inline libmscore/sym.cpp:3286]: same here

File: mscore/palette/palettetree.cpp [hunk 3]
```diff
@@ -122,6 +122,7 @@ const char* PaletteCell::translationContext() const
       switch (type) {
             case ElementType::ACCIDENTAL:
             case ElementType::ARTICULATION:
+            case ElementType::BAR_LINE:
```
Comment (Jojo-Schmitz) [inline mscore/palette/palettetree.cpp:125]: using `symUserNames' context for barlines too

**Comment by anatoly-os** [comment]
Seems like this PR is not needed for master due to the merged #6119. @Jojo-Schmitz what do you think?

**Comment by Jojo-Schmitz** [comment]
Let me check that again... last time I looked I think it was still needed

**Comment by Jojo-Schmitz** [comment]
Yes, you're right, sorry for the noise

**Pull request merged**